unstructured/partition/text.py

```diff
@@ -60,7 +57,6 @@ def partition_text(
     chunking_strategy: Optional[str] = None,
```

Comment:
Should this kwarg go away like it did for `pptx`?

Reply:
Ah, good catch, yes, I'll do that ...